Project-Aurora/Project-Aurora/Profiles/Witcher3/GameEvent_Witcher3.cs

```diff
@@ -66,79 +63,64 @@ public override void ResetGameState()
             _game_state = new GameState_Witcher3();
         }
 
-        public new bool IsEnabled
-        {
-            get { return this.Application.Settings.IsEnabled && isInitialized; }
-        }
-
         public override void UpdateLights(EffectFrame frame)
         {
             Queue<EffectLayer> layers = new Queue<EffectLayer>();
 
             if (File.Exists(dataPath))
             {
-                if(configContent != null)
+                if (configContent != null)
                 {
-                    var signRes = _configRegex.Match(configContent);
```

Comment:
Possibly want to be careful with using a regex each update as they may not always be as fast as other methods. While not being able to see what a config looks like, I think for this instance you could probably use `String.IndexOf` method to achieve the same.

Though if performance is fine currently, then this is probably just nit-picking.

Reply:
IndexOf is indeed useful for this. I rewrote most of the parsing in the latest commit and it's not using regular expressions anymore. Should also be less repetitive and more readable.

As for performance, i played a couple of minutes and it seemed a little better than before but it's probably placebo. In any case it's an improvement as it doesnt have to do as much processing each tick.